vhs.go

```diff
@@ -89,7 +89,11 @@ func New() VHS {
 
 	opts := DefaultVHSOptions()
 	path, _ := launcher.LookPath()
-	u := launcher.New().Leakless(false).Bin(path).MustLaunch()
+	launcher := launcher.New().Leakless(false).Bin(path)
+	if os.Getenv("VHS_NO_SANDBOX") != "" {
+		launcher = launcher.NoSandbox(true)
+	}
+	u := launcher.MustLaunch()
```

Comment:
I _think_ you can do:
```suggestion
    noSandbox := os.Getenv("VHS_NO_SANDBOX") != ""
	u := launcher.New().Leakless(false).Bin(path).NoSandbox(noSandbox).MustLaunch()
```

Reply:
yup works perfectly fine.